authenticated-digital-emblem-scoping.md

```diff
@@ -2,7 +2,27 @@
 
 ## Background
 
-During armed conflicts, infrastructure marked with certain distinctive emblems or signs, including the Red Cross, Red Crescent, and Red Crystal, are given special protections under International Humanitarian Law (IHL) and cannot be attacked under almost all circumstances. As conflicts now involve cyberattacks against digital resources, clear authenticated communication of equivalent digital emblems for digital assets can facilitate compliance with IHL and foster stronger protection of medical and humanitarian resources. Extending digital emblems must be accessible through well-known, consensus mechanisms that are respected, verifiable, and accountable against misuse.
+During armed conflict, certain humanitarian infrastructure enjoys special protection under International Humanitarian Law (IHL) and must be respected and protected by all parties to the conflict.
+IHL defines a number of physical emblems, including the Red Cross, Red Crescent, and Red Crystal, that signal this protection in the physical world.
+However, there is no way to signal this protection in the digital world.
+
+As conflicts now increasingly take place in the digital world, signalling protection using equivalent digital emblems facilitates the protection of medical and humanitarian digital assets.
+Digital emblems must be accessible through well-known mechanisms and must be respected, verifiable, and accountable against misuse.
+
+## Goals
+
+The working seeks to develop a mechanism that:
+
+- can signal that a digital asset enjoys protection under IHL using network interfaces;
+- parties to a conflict are willing to use; such parties typically intend to engage in offensive operations and, for example, will not tolerate signalling this intention.
```

Comment:
Sentence after semicolon = covert inspection (but generalized). I, on the other hand, do not quite follow what you want to say with the addition in parentheses 😁 Could you elaborate? I think the "willing to use" part is not just for compliance with existing IHL. Actually, your addition might warrant its own bullet point.

Reply:
But I certainly agree that my sentence after semicolon is quite vaguely put.

Reply:
Ohh, I see, I was confused by signal meaning "warn the target you are verifying they do not have an emblem" versus "informing others I have an emblem". Now I see we are indeed making two different points, and I agree about splitting them. How about this: 

```suggestion
- parties to a conflict are willing to use (so that checking for an emblem's existence does not require the party doing the checking to reveal their actions to the other party); 
- has design agreement with the necessary legal guardians of the in-scope IHL emblems (so that the working group's produced work can actually be used for its intended purpose);
```